PennMobile/Dining/Model/DiningVenue+Extensions.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  DiningVenue+UIExtensions.swift
+//  PennMobile
+//
+//  Created by Dominic Holmes on 10/21/19.
+//  Copyright © 2019 PennLabs. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - VenueType
+extension DiningVenue.VenueType {
+    var fullDisplayName: String {
+        switch self {
+        case .dining: return "Campus Dining Hall"
+        case .retail: return "Campus Retail Dining"
+        case .unknown: return "Other"
+        }
+    }
+}
+
+// MARK: - DiningVenue
+extension DiningVenue {
+    
+    // MARK: - Defaults
+    static let defaultVenueIds: [Int] = [593, 636, 1442]
```

Comment:
What are these for? I thought we were aiming for a solution with zero data on the device.

Reply:
These are ids for commons, hill, lch.

I figured we should still have a way to prioritize commons / provide defaults on the home page. We could also include this in our API endpoint if we really want to remove it from the code.

Reply:
They're used to display the default selected dining halls